templates/user/identityForm.tpl

```diff
@@ -80,3 +92,8 @@
 
 	{fbvFormButtons hideCancel=true submitText="common.save"}
 </form>
+
+
+<script>
```

Comment:
Did you mean to have this empty script tag here?

Reply:
should be removed